packages/render-fidelity-tools/src/artifact-creator.ts

```diff
@@ -56,7 +56,7 @@ export class ArtifactCreator {
 
     for (const scenarioBase of scenarios) {
       const scenarioName = scenarioBase.name;
-      const scenario = this[$configReader].scenarioConfig(scenarioName)!;
+      const scenario = this[$configReader].scenarioConfig(scenarioName)!;  // ??
```

Comment:
Do you have a question for me?

Reply:
just ignore it, i didn't mean to commit it. i don't understand the syntax at first so I leave a mark here. I have already learned what does this mean. I will remove this in the new push